internal/sign/helper.go

```diff
@@ -2,12 +2,13 @@ package sign
 
 import (
 	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+	"strings"
 )
 
 //go:generate mockgen -source=helper.go -package=sign -destination=mock_helper.go
 
 type Helper interface {
-	GetRelevantSign(modSpec kmmv1beta1.ModuleSpec, km kmmv1beta1.KernelMapping) *kmmv1beta1.Sign
+	GetRelevantSign(modSpec kmmv1beta1.ModuleSpec, km kmmv1beta1.KernelMapping, macros map[string]string) *kmmv1beta1.Sign
```

Comment:
Why do we need to support a `map[string]string` of changes if all we ever do is just changing `KERNEL_VERSION`?

Reply:
Ack. In that case, I will comment on the implementation of the map.